readme/BREAKING_CHANGES.markdown

```diff
@@ -340,4 +340,45 @@ By moving minification of frontend resources from run-time to build-time
 we reduce server load and gain access to the latest minification
 technologies available within the frontend ecosystem.
 
+---------------------------------------
+
+### SoyPortlet is no longer available
+- **Date:** 2020-Dec-9
+- **JIRA Ticket:** [LPS-122955](https://issues.liferay.com/browse/LPS-122955)
+
+#### What changed?
+
+The class `SoyPortlet` used to implement Portlet whose views are backed by
+Closure Templates (Soy) has been removed and is no longer available.
+
+#### Who is affected?
+
+Anyone using `SoyPortlet` as a base for their portlet developments.
+
+#### How should I update my code?
+
+We heavily recommend re-writing your Soy portlets using either a well stablished
+architecture such as MCVPortlet using JSPs or a particular frontend framework of
```

Comment:
Should `MCVPortlet` be in backticks? Also, should it be **MVC**Portlet?

Reply:
Nice catch!